README.md

```diff
@@ -1,18 +1,49 @@
 # vets-api-clients
 
-This repository contains resources for accessing vets-api as a third-party
-client. It includes API specifications, sample clients, and general
-access information.
+Welcome to the public-facing repository for **VA Lighthouse APIs**.  For the most up-to-date API documentation, please visit [developer.va.gov/explore](https://developer.va.gov/explore).
```

Comment:
Could we make sure we refer to the Department of Veterans Affairs to just to be clear what VA stands for?
For example: Welcome to the Department of Veterans Affairs (VA) public-facing repository...

Reply:
I'm with you, did that originally and then second-guessed it!

Reply:
Calling this out as public facing seems targeted inward (to those that know there are private repos) .. to the audience of this repo, calling it public facing may actually complicate things.